content/documentation/PinePhone_Pro/Software/Multi-distribution_image.adoc

```diff
@@ -0,0 +1,291 @@
+---
+title: "Multi-distribution image"
+draft: false
+menu:
+  docs:
+    title:
+    parent: "PinePhone_Pro/Software"
+    identifier: "PinePhone_Pro/Software/Multi-distribution_image"
+    weight: 99
+---
+
+:toc:
+
+This article explains how to install a multiple linux distribution enviroment on your PinePhone Pro that is bootable from a microSD card.
+
+== Installing rk2aw
+
+Make sure your phone has the link:/documentation/PinePhone_Pro/Software/Bootloaders/#rk2aw[rk2aw pre-loader] installed to SPI flash. PinePhone Pro models ordered after November 2023 are shipped with rk2aw pre-installed. Otherwise, follow the instructions in this section to install it.
+
+TIP: The multi-boot image _can_ function without rk2aw, but you may need to hold down the **RE** button during boot to disable the SPI flash memory.
```

Comment:
Holding the RE button disables the eMMC as well (not just the SPI). It might be better to say "... to force your phone to boot from the microSD card."